src/VirtoCommerce.InventoryModule.Data/VirtoCommerce.InventoryModule.Data.csproj

```diff
@@ -10,19 +10,8 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Hangfire" Version="1.7.9" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.2" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="3.1.2">
-      <PrivateAssets>all</PrivateAssets>
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-    </PackageReference>
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="3.1.2">
-      <PrivateAssets>all</PrivateAssets>
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-    </PackageReference>
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="3.1.2" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="3.1.2" />
     <PackageReference Include="VirtoCommerce.CoreModule.Core" Version="3.1.0" />
-    <PackageReference Include="VirtoCommerce.Platform.Data" Version="3.5.0" />
+    <PackageReference Include="VirtoCommerce.Platform.Data" Version="3.32.0" />
```

Comment:
I prefer to not remove explicit references. Transitive references allow us to do this, but in that case, removing some of these refs from Platform.Data will break this module.